src/main/java/bio/terra/service/auth/iam/IamResourceType.java

```diff
@@ -36,4 +37,32 @@ static IamResourceType fromValue(String text) {
     }
     return null;
   }
+
+  public static IamResourceType toIamResourceType(IamResourceTypeEnum apiEnum) {
+    if (apiEnum == IamResourceTypeEnum.SPEND_PROFILE) {
```

Comment:
Wouldn't this be handled by `toIamResourceType(String stringResourceType)` already, since the enum name is `SPEND_PROFILE` in both cases?

Reply:
Unfortunately, it does not because of the variance in the associated value("spend-profile").  There's an assert in IamServiceTypeTest that will fail if this code is removed.